libs/rttrConfig/src/files.h

```diff
@@ -35,6 +35,7 @@ namespace folders {
     constexpr auto mapsPlayed = "<RTTR_USERDATA>/MAPS"; // downloaded maps
     constexpr auto mapsRttr = "<RTTR_RTTR>/MAPS/NEW";   // maps added by RTTR
     constexpr auto mapsSea = "<RTTR_RTTR>/MAPS/SEA";    // seafaring maps
+    constexpr auto campaigns = "<RTTR_RTTR>/CAMPAIGNS"; // contains one folder for each campaign
```

Comment:
Should we maybe have a user folder for campaigns, e.g. `<RTTR_USERDATA>/campaigns`? The `RTTR_RTTR` folder is in the install dir and might not be user-writable.

Reply:
perhaps load both?
the first one in `RTTR_RTTR` for our campaigns (aka romans, ...) and for users campaigns one in `RTTR_USERDATA` - perhaps like we have with the map selection with categories?

Reply:
Added the additional search path for user defined campaigns.